src/aspect-meta-model/characteristic/default-either.ts

```diff
@@ -28,37 +31,29 @@ export interface Either extends Characteristic {
 }
 
 export class DefaultEither extends DefaultCharacteristic implements Either {
-    constructor(
-        metaModelVersion: string,
-        aspectModelUrn: string,
-        name: string,
-        private _left: Characteristic,
-        private _right: Characteristic
-    ) {
-        super(metaModelVersion, aspectModelUrn, name);
-    }
-
-    public set left(value: Characteristic) {
-        this._left = value;
-    }
+    left: Characteristic;
+    right: Characteristic;
 
-    public get left(): Characteristic {
-        return this._left;
+    constructor(props: EitherProps) {
+        super(props);
+        this.left = props.left;
+        this.right = props.right;
     }
 
-    public set right(value: Characteristic) {
-        this._right = value;
+    getLeft(): Characteristic {
```

Comment:
Can we please change that in general back to the new and more accepted approach to implement getter and setter (see also https://effectivetypescript.com/2023/12/31/getters-setters/). It was back then as I started to get more acceptance by a possible community (it in general it is also a way to use from my perspective).